packages/gatsby-source-prismic/src/constants.ts

```diff
@@ -12,6 +12,15 @@ export const GLOBAL_TYPE_PREFIX = 'Prismic'
  */
 export const PRISMIC_API_NON_DATA_FIELDS = ['uid']
 
+/**
+ * Endpoint used to fetch custom type JSON schemas from Prismic's Custom Type
+ * API.
+ *
+ * @see https://prismic.io/docs/technologies/custom-types-api
+ */
+export const PRISMIC_CUSTOM_TYPES_API_ENDPOINT =
+  'https://customtypes.prismic.io/customtypes'
```

Comment:
occasionally we'll run projects against wroom.io and wroom.test (usually to recrate bugs)

if this goes clients side leave it as is and we'll work with a local version of the plugin :)

or maybe use `process?.env?.PRISMIC_CUSTOM_TYPES_API_ENDPOINT || `'https://customtypes.prismic.io/customtypes'

Reply:
Good to know! Maybe we can add another plugin option like `apiEndpoint` - it defaults to the default `prismic.io` URL, but can be overridden to anything.

I can add `customTypesApiEndpoint` so it can be configured either through a string or env variable.